lstchain/scripts/lstchain_mc_sensitivity.py

```diff
@@ -6,73 +6,81 @@
 import numpy as np
 import argparse
 import ctaplot
+from lstchain.visualization import plot_dl2
+from lstchain.reco import utils
 import seaborn as sns
 from lstchain.io import read_simu_info_merged_hdf5
 
```

Comment:
`from lstchain.mc import sensitivty` is required there no?

Reply:
it is defined in line 1